packages/repeater/README.md

```diff
@@ -0,0 +1,82 @@
+# @secbox/repeater
+
+Package to manage repeaters and their lifecycle.
+
+Repeaters are mandatory for scanning targets on a local network.
+More info about [repeaters](https://docs.brightsec.com/docs/on-premises-repeater-local-agent).
+
+## Setup
+
+```bash
+npm i -s @secbox/repeater
+```
+
+## Usage
+
+`RepeaterFactory` exported by this package can be constructed with [`Configuration` instance](https://github.com/NeuraLegion/secbox-sdk-js/tree/master/packages/core#configuration) as constructor argument.
+
+It has a method `createRepeater()` which returns a `Repeater` instance.
```

Comment:
Let's add an example of how to create a dedicated repeater

Reply:
please add more details on "dedicated"

Reply:
Let me update the docs)

Reply:
See [f35dac7](https://github.com/NeuraLegion/secbox-sdk-js/pull/28/commits/f35dac7b5833e7118160fd7e50bc13428b56506a)